docs/articles/testing/multi-node-testing-old.md

```diff
@@ -0,0 +1,286 @@
+
+```csharp
+RunOn(() => {
+    TestConductor.Exit(_config.Third, 0).Wait();
+}, _config.First);
+```
+
+Once a node has exited the test, it will no longer be able to wait on `EnterBarrier` calls and the multi-node test runner will not try to collect any data from that node from that point onward.
+
+## Running Multi-Node Tests
+
+Once you've coded your multi-node tests and compiled them, it's now time to run them. Akka.NET ships a custom XUnit2 runner that it uses to create the simulated networks and clusters and you will need to install that via NuGet in order to run your tests:
+
+```console
+PS> nuget.exe Install-Package Akka.MultiNodeTestRunner -NoVersion
+```
+
+This will install the [Akka.MultiNodeTestRunner NuGet package](https://www.nuget.org/packages/Akka.MultiNodeTestRunner) with the following directory and file structure:
+
+```text
+root/akka.multinodetestrunner
+root/akka.multinodetestrunner/lib/net452/Akka.MultiNodeTestRunner.exe
+root/akka.multinodetestrunner/lib/netcoreapp1.1/Akka.MultiNodeTestRunner.dll
+```
+
+Depending on what framework you're building your application against, you'll want to pick the appropriate tool (.NET Framework or .NET Core.)
+
+Next, we have to pass in our command-line arguments to the MNTR:
+
+```console
+Akka.MultiNodeTestRunner.exe [path to assembly] [-Dmultinode.enable-filesink=on] [-Dmultinode.output-directory={dir path}] [-Dmultinode.spec={spec name}]
+```
+
+We strongly recommend setting the `-Dmultinode.output-directory={dir path}` directory to some local folder you can access, as the multi-node test runner will emit:
+
+1. An output file for the entire test run of the DLL and
+2. For each individual spec, a subfolder that contains logs pertaining to the original node.
+
+_Hint:_ Each test run will append new log entries to output files.
+If this is not desired, you can pass `-Dmultinode.clear-output=1` option to delete output folder before MNTR will run tests.
+
+If you're lost and need more examples, please explore the Akka.NET source code and take a look at some of the MNTR output produced by our CI system on any open pull request.
+
+## Debugging Failed Tests
+
+As already mentioned, after each spec is finished, test runner wil emit log files for it to output directory subfolder with the full name of the spec.
+In this folder you will find individual logs for each node (named according to the roles they were assigned), and `aggregated.txt` file,
+which contains all nodes logs aggregated into single timeline.
+
+Also, `FAILED_SPECS_LOGS` subdirectory will be generated. If any of your specs failed, this folder will contain aggregated logs for each spec -
+basically, the same `aggregated.txt` files but with their spec's names. This is a good place to get a full picture of what has failed and why.
+
+Also, `-Dmultinode.failed-specs-directory={failed spec dir}` option could be used to override `FAILED_SPECS_LOGS` name.
```

Comment:
Just curious, why do we need the old one?

Reply:
Because we need to support the old MNTR no matter what, even when we deprecate it. Someone might still be using it and it doesn't cost us anything to keep the documentation up.